File: JavaSort/SelectionSort.java
import java.util.Arrays;

public class SelectionSort {
    public static void main(String[] args) {
        int[] arr = {2,6,29,17,3,91,70,61,20};

        for (int i = 0; i < arr.length-1; i++) {
            int min =i;
            for (int j = i+1; j < arr.length; j++) {
                if (arr[j]< arr[min]){
                    min=j;
                }
            }
            int temp = arr[i];
            arr[i]=arr[min];
            arr[min]= temp;
        }
        System.out.println("The Sorted Array is: " + Arrays.toString(arr));
    }
}

File: PythonSort/selectionSort.py
def selection_sort(array):
    for i in range(len(array)):
        min_index = i
        for j in range(i+1, len(array)):
            if array[j] < array[min_index]:
                min_index = j
        
        array[i], array[min_index] = array[min_index], array[i]

    return array

arr = [64,81,97,18,10,891,72,110,69,21,7,-56,32,-7]

sorted_arr = selection_sort(arr)

print("The sorted array is: ", sorted_arr)